app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim

```diff
@@ -54,6 +54,7 @@ section.Configuration class=('Configuration--is-promoted' unless deployment_opti
           dl.u-dl
             dt.u-dl-term URL
             dd.u-dl-definition = @show_presenter.staging_proxy_endpoint
+        - unless @proxy.service.oauth?
```

Comment:
Now the code looks like this:
```ruby
        - if @show_presenter.any_sandbox_configs?
          dl.u-dl
            dt.u-dl-term URL
            dd.u-dl-definition = @show_presenter.staging_proxy_endpoint
        - unless @proxy.service.oauth?
            dt.u-dl-term Example curl for testing
            dd.u-dl-definition
              = api_test_curl(@proxy)
              - unless @service.cinstances.any?
                br
                br
                hr
                em
                  => t('api.integrations.proxy.curl.no_application')
                  - unless @service.application_plans.any?
                    = link_to t('api.integrations.proxy.curl.no_application_plan'), admin_service_application_plans_path(@service)
            dt.u-dl-term Version
            dd.u-dl-definition
              ' v.
              = @show_presenter.last_sandbox_config.version
            dt.u-dl-term Promote
            dd.u-dl-definition.Configuration-promote-button
              = semantic_form_for @proxy, url: promote_to_production_admin_service_integration_path(@service, anchor: 'production') do |f|
                = f.button *promote_to_production_button_options(@show_presenter)
        - else
          | no configuration has been saved for APIcast staging yet
```

In a more 'visible' way:
```ruby
        - if @show_presenter.any_sandbox_configs?
          # ...
        - unless @proxy.service.oauth?
          # ...
        - else
          | no configuration has been saved for APIcast staging yet
```

Imo this is not correct... that unless should not have the same indentation as the other 2 conditions... and it should be:
```ruby
        - if @show_presenter.any_sandbox_configs?
          # ...
          - unless @proxy.service.oauth?
            # ...
        - else
          | no configuration has been saved for APIcast staging yet
```

Reply:
Which exactly would be:
```ruby
        - if @show_presenter.any_sandbox_configs?
          dl.u-dl
            dt.u-dl-term URL
            dd.u-dl-definition = @show_presenter.staging_proxy_endpoint
            - unless @proxy.service.oauth?
              dt.u-dl-term Example curl for testing
              dd.u-dl-definition
                = api_test_curl(@proxy)
                - unless @service.cinstances.any?
                  br
                  br
                  hr
                  em
                    => t('api.integrations.proxy.curl.no_application')
                    - unless @service.application_plans.any?
                      = link_to t('api.integrations.proxy.curl.no_application_plan'), admin_service_application_plans_path(@service)
            dt.u-dl-term Version
            dd.u-dl-definition
              ' v.
              = @show_presenter.last_sandbox_config.version
            dt.u-dl-term Promote
            dd.u-dl-definition.Configuration-promote-button
              = semantic_form_for @proxy, url: promote_to_production_admin_service_integration_path(@service, anchor: 'production') do |f|
                = f.button *promote_to_production_button_options(@show_presenter)
        - else
          | no configuration has been saved for APIcast staging yet
```